editor.planx.uk/src/@planx/components/MapAndLabel/Public/MapAndLabel.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import { Meta, StoryObj } from "@storybook/react";
+import React from "react";
+
+import Wrapper from "../../fixtures/Wrapper";
+import Editor from "../Editor";
+import { Trees } from "../schemas/Trees";
+import { Presentational as Public, PresentationalProps } from ".";
+
+const meta = {
+  title: "PlanX Components/MapAndLabel",
+  component: Public,
+} satisfies Meta<typeof Public>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+const props: PresentationalProps = {
+  title: "Map and label the works to trees for this property",
+  description: "Add one or many trees below",
+  schemaName: "Trees",
+  fn: "MockFn",
+  schema: Trees,
+  basemap: "OSM",
```

Comment:
When set to `MapboxSatellite` I get permission errors (works fine when running the application).

I think we need to allow `localhost:6006` (Storybook) on MapBox for this to work.

Reply:
Good catch! Added `http://localhost:6006` to allowed URLs for our "Planx local dev" token (ends with `NW1Q`) if you can check again? Hosted Storybook should be properly captured under the second key for our planx subdomains (token ends with `au8A`). 

These are are available named tokens for future ref:
![Screenshot from 2024-08-28 20-45-40](https://github.com/user-attachments/assets/d8b167ca-f4f5-458c-b416-9d5566326918)

Reply:
Still getting this issue locally, the key I'm using does match what you've provided above. Not a silly issue with a trailing whitespace or slash or something? (seems unlikely!)

Either way, I don't think this is necessarily blocking just now but would be nice to resolve.

Reply:
Will double check this one with fresh eyes this morning & test locally - sure we can get it sorted going forward :+1: